style.css

```diff
@@ -0,0 +1,117 @@
+
+* {
+    box-sizing: border-box;
+}
+
+body {
+    margin: 0;
+    overflow: auto;
+}
+
+.logo {
+    height: 50%;
+    width: 10%;
+    display: block;
+    border: solid black 1px;
+}
+
+h1 {
+    margin: 0;
+}
+
+#one {
+    border-top: solid black 1px;
+    overflow: auto;
+}
+
+#two {
+    border-top: solid black 1px;
+    overflow: auto;
+}
+
+#three {
+    border-top: solid black 1px;
+}
+
+#four {
+    border-top: solid black 1px;
+}
+
+nav {
+    position: fixed;
+    top: 0;
+    background: paleturquoise;
+    height: 8%;
+    opacity: 0.5;
+}
+
+ul{
+    list-style-type: none;
+    text-align: center;
+    padding: 1% 0% 0% 0%;
+}
+li {
+    display: inline;
+    padding: 0% 5% 0% 5%;
+}
+
+.container {
+    width: 50%;
+    margin: auto;
+    overflow: auto;
+    border: solid black 1px;
+}
+
+.about-me-text {
+    text-align: left;
+    width: 500px;
```

Comment:
needs to be a % as its currently cutting text off at smaller screen widths

Reply:
Done